app/uk/gov/hmrc/traderservices/views/ImportQuestionsReasonView.scala.html

```diff
@@ -22,20 +22,31 @@
 )
 @(form: Form[String], postAction: Call, backLink: Call)(implicit request: Request[_], messages: Messages)
 
+@inputRadioHint = {
+@html.p {
+    @messages("form.import-questions.reason-text.hint.description1")
+}
+@html.ul(Seq("form.import-questions.reason-text.hint.list.cancel",
+    "form.import-questions.reason-text.hint.list.withdraw",
```

Comment:
Sorry, I didn't flag it here before but I think we need to remove this also?

Reply:
yes that's right